enhancements/baremetal-IPI/baremetal.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Neat-Enhancement-Idea
```

Comment:
It is a neat idea, but I think you want the actual doc title here? ;-)

Reply:
If you say so, @dhellmann :-). Needed to check if it had to be a hyphenated string and didn't get back to it.